Doc/conf.py

```diff
@@ -254,6 +254,15 @@
 # Options for the link checker
 # ----------------------------
 
+linkcheck_allowed_redirects = {
+    # bpo-NNNN -> BPO -> GH Issues
+    r'https://bugs.python.org/issue\?@action=redirect&bpo=\d+': 'https://github.com/python/cpython/issues/\d+',
+    # GH-NNNN used to refer to pull requests
+    r'https://github.com/python/cpython/issues/\d+': 'https://github.com/python/cpython/pull/\d+',
+    # :source:`something` linking files in the repository
+    r'https://github.com/python/cpython/tree/.*': 'https://github.com/python/cpython/blob/.*'
+}
+
 # Ignore certain URLs.
 linkcheck_ignore = [r'https://bugs.python.org/(issue)?\d+']
```

Comment:
Is this entry still needed?

Reply:
Looks like it isn't needed, I'll remove it.

There are a few direct links to BPO (I.e. not using bpo-NNNNN) in the docs, and linkcheck returns 'ok' to these links.

For the record, this line was added in 0ef49667, although I'm not sure what issue  BPO links were causing.

Reply:
Done